### DISCUSSION TITLE
For the last few days the server seems to "crash"

### DISCUSSION THREAD

**Discussion by fugufish** [community/community#3826] [Codespaces]
For the last few days my code spaces seem to lose connectivity and I cannot reconnect to the space unless I shut it down and start it back up again. It will work for a bit and then the problem repeats itself. I have created new codespaces with the same result. I checked the memory and drive space and both seem fine.

**Comment by matthewisabel** [comment]
Thanks for reporting. Do you happen to be using a custom devcontainer?
